kotlin/find-address/src/main/java/com/esri/arcgisruntime/sample/findaddress/MainActivity.kt

```diff
@@ -0,0 +1,342 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.findaddress
+
+import android.database.MatrixCursor
+import android.graphics.Color
+import android.graphics.drawable.BitmapDrawable
+import android.os.Bundle
+import android.provider.BaseColumns
+import android.util.Log
+import android.view.MotionEvent
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import androidx.appcompat.widget.SearchView
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.cursoradapter.widget.SimpleCursorAdapter
+import com.esri.arcgisruntime.concurrent.ListenableFuture
+import com.esri.arcgisruntime.loadable.LoadStatus
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.Viewpoint
+import com.esri.arcgisruntime.mapping.view.Callout
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import com.esri.arcgisruntime.mapping.view.Graphic
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay
+import com.esri.arcgisruntime.mapping.view.IdentifyGraphicsOverlayResult
+import com.esri.arcgisruntime.symbology.PictureMarkerSymbol
+import com.esri.arcgisruntime.tasks.geocode.GeocodeParameters
+import com.esri.arcgisruntime.tasks.geocode.GeocodeResult
+import com.esri.arcgisruntime.tasks.geocode.LocatorTask
+import kotlinx.android.synthetic.main.activity_main.*
+import java.util.concurrent.ExecutionException
+import kotlin.math.roundToInt
+
+class MainActivity : AppCompatActivity() {
+
+  private val TAG: String = MainActivity::class.java.simpleName
+  private var callout: Callout? = null
+  private var addressGeocodeParameters: GeocodeParameters? = null
+  // create a picture marker symbol
+  private val pinSourceSymbol: PictureMarkerSymbol? by lazy { createPinSymbol() }
+  // create a locator task from an online service
+  private val locatorTask: LocatorTask by lazy { LocatorTask(getString(R.string.locator_task_uri)) }
+  // create a new Graphics Overlay
+  private val graphicsOverlay: GraphicsOverlay by lazy { GraphicsOverlay() }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a map with the streets vector basemap type
+    val topographicBasemap = ArcGISMap(Basemap.createStreetsVector())
```

Comment:
```suggestion
    val topographicMap= ArcGISMap(Basemap.createStreetsVector())
```

Maybe best to not call this object a `Basemap` when thats another name in our API